test/chat_api_web/channels/notification_channel_test.exs

```diff
@@ -42,4 +44,44 @@ defmodule ChatApiWeb.NotificationChannelTest do
     broadcast_from!(socket, "broadcast", %{"some" => "data"})
     assert_push "broadcast", %{"some" => "data"}
   end
+
+  test "conversation first replied at is updated", %{
```

Comment:
nice! we should probably also test that this only happens for the first reply message